mlflow/server/js/src/app.tsx

```diff
@@ -35,6 +36,7 @@ export function MLFlowRoot() {
     <IntlProvider locale={locale} messages={messages}>
       <Provider store={store}>
         <DesignSystemContainer isDarkTheme={isDarkTheme}>
+          <ApplyGlobalStyles />
```

Comment:
@hubertzub-db is this the correct usage?

Reply:
@daniellok-db please just add it to `<MlflowRouter>` and I think that's it :)

Reply:
Does it need to be added there as well? I was thinking this would be enough as it's the parent component to both `<MlflowRouter>` and `<App>`

Reply:
Ahhh my bad! `app.tsx` != `App.tsx` 😅 
It's perfect then :). However, we have to keep it mind to retest it after reimporting to Databricks  because we will have two `<ApplyGlobalStyles />` then (one from monolith, one from mlflow) - but I don't think there will by any issue

Reply:
Sounds good! I'll go ahead and merge to the feature branch!